<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>859</width>
    <height>848</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QTextEdit {
    border: 2px solid green;
    border-radius: 10px;
    padding: 5px;
    background-color: white;
    selection-background-color: lightgray;
}

QTextEdit:focus {
    border: 2px solid #003366; /* Màu xanh đen */
}
QLineEdit {
    border: 2px solid green;
    border-radius: 10px;
    padding: 8px 15px;
    background-color: white;
    selection-background-color: lightgray;
}

QLineEdit:focus {
    border: 2px solid #003366; /* Màu xanh đen */
}

QPushButton {
    background-color: #0099FF; /* Màu xanh da trời */
    border: none;
    border-radius: 8px;
    padding: 8px 16px;
    color: white;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #007ACC; /* Màu xanh đậm hơn khi hover */
}

QPushButton:pressed {
    background-color: #005F99; /* Màu xanh đậm hơn khi bấm */
}
QProgressBar {
                border: 2px solid #008000;  /* Viền xanh lá cây đậm */
                border-radius: 5px;
                background-color: white;
                text-align: center;
            }
            QProgressBar::chunk {
                background-color: #90EE90; /* Tiến trình xanh lá cây nhạt */
                width: 10px;
            }</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="verticalWidget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="leftMargin">
        <number>40</number>
       </property>
       <property name="topMargin">
        <number>40</number>
       </property>
       <property name="rightMargin">
        <number>40</number>
       </property>
       <property name="bottomMargin">
        <number>40</number>
       </property>
       <item alignment="Qt::AlignTop">
        <widget class="QWidget" name="horizontalWidget_5" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Nhập đường đãn: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txtUrl">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="placeholderText">
             <string>https://www.youtube.com/</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cbbSocial">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">
QComboBox {
    border: 2px solid red;
    border-radius: 10px;
    padding: 8px 15px;
    background-color: white;
    selection-background-color: redgray;
}

QComboBox:focus {
    border: 2px solid #003366; /* Xanh đen */
}

QComboBox::drop-down {
    border: none;
    width: 20px;
    background: transparent;
}

QComboBox::down-arrow {
    image: url(down_arrow.png); /* Đổi icon mũi tên nếu cần */
    width: 12px;
    height: 12px;
}

QComboBox QAbstractItemView {
    border: 1px solid #003366;
    selection-background-color: #0099FF; /* Xanh da trời khi chọn */
    background: white;
    border-radius: 5px;
}
</string>
            </property>
            <item>
             <property name="text">
              <string>Youtube</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Douyin</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Weibo</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Tiktok</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item alignment="Qt::AlignTop">
        <widget class="QWidget" name="horizontalWidget_4" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Lựa chọn thư mục</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txtFolderSave">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="placeholderText">
             <string>c:/</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnSelectFolder">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Lựa chọn</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item alignment="Qt::AlignLeft">
        <widget class="QWidget" name="horizontalWidget_3" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Chất lượng video Youtube: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cbbQuatity">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">
QComboBox {
    border: 2px solid red;
    border-radius: 10px;
    padding: 8px 15px;
    background-color: white;
    selection-background-color: lightgray;
}

QComboBox:focus {
    border: 2px solid #003366; /* Xanh đen */
}

QComboBox::drop-down {
    border: none;
    width: 20px;
    background: transparent;
}

QComboBox::down-arrow {
    image: url(down_arrow.png); /* Đổi icon mũi tên nếu cần */
    width: 12px;
    height: 12px;
}

QComboBox QAbstractItemView {
    border: 1px solid #003366;
    selection-background-color: #0099FF; /* Xanh da trời khi chọn */
    background: white;
    border-radius: 5px;
}
</string>
            </property>
            <item>
             <property name="text">
              <string>bestvideo+bestaudio</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>bestvideo</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>bestaudio</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>worstvideo</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>worstaudio</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item alignment="Qt::AlignLeft">
        <widget class="QWidget" name="horizontalWidget_2" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Loại đường dẫn link: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cbbTypeLink">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">
QComboBox {
    border: 2px solid red;
    border-radius: 10px;
    padding: 8px 15px;
    background-color: white;
    selection-background-color: lightgray;
}

QComboBox:focus {
    border: 2px solid #003366; /* Xanh đen */
}

QComboBox::drop-down {
    border: none;
    width: 20px;
    background: transparent;
}

QComboBox::down-arrow {
    image: url(down_arrow.png); /* Đổi icon mũi tên nếu cần */
    width: 12px;
    height: 12px;
}

QComboBox QAbstractItemView {
    border: 1px solid #003366;
    selection-background-color: #0099FF; /* Xanh da trời khi chọn */
    background: white;
    border-radius: 5px;
}
</string>
            </property>
            <item>
             <property name="text">
              <string>Link</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>User</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">
QComboBox {
    border: 2px solid red;
    border-radius: 10px;
    padding: 8px 15px;
    background-color: white;
    selection-background-color: lightgray;
}

QComboBox:focus {
    border: 2px solid #003366; /* Xanh đen */
}

QComboBox::drop-down {
    border: none;
    width: 20px;
    background: transparent;
}

QComboBox::down-arrow {
    image: url(down_arrow.png); /* Đổi icon mũi tên nếu cần */
    width: 12px;
    height: 12px;
}

QComboBox QAbstractItemView {
    border: 1px solid #003366;
    selection-background-color: #0099FF; /* Xanh da trời khi chọn */
    background: white;
    border-radius: 5px;
}
</string>
            </property>
            <item>
             <property name="text">
              <string>video</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>short</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item alignment="Qt::AlignLeft">
        <widget class="QWidget" name="horizontalWidget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Loại video Douyin: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cbbTypeDownloadDouyin">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">
QComboBox {
    border: 2px solid red;
    border-radius: 10px;
    padding: 8px 15px;
    background-color: white;
    selection-background-color: lightgray;
}

QComboBox:focus {
    border: 2px solid #003366; /* Xanh đen */
}

QComboBox::drop-down {
    border: none;
    width: 20px;
    background: transparent;
}

QComboBox::down-arrow {
    image: url(down_arrow.png); /* Đổi icon mũi tên nếu cần */
    width: 12px;
    height: 12px;
}

QComboBox QAbstractItemView {
    border: 1px solid #003366;
    selection-background-color: #0099FF; /* Xanh da trời khi chọn */
    background: white;
    border-radius: 5px;
}
</string>
            </property>
            <item>
             <property name="text">
              <string>all</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>video</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>picture</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">margin: 1px 15px</string>
            </property>
            <property name="text">
             <string>Số lượng video: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txtQuantity"/>
          </item>
          <item>
           <widget class="QCheckBox" name="cbAll">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">margin: 1px 30px</string>
            </property>
            <property name="text">
             <string>Tải toàn bộ</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="txtLog"/>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>TIẾN TRÌNH TỔNG</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="pbMain">
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbChill">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Tiến trình phụ</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="pbChill">
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="leftMargin">
          <number>7</number>
         </property>
         <property name="topMargin">
          <number>7</number>
         </property>
         <property name="rightMargin">
          <number>7</number>
         </property>
         <property name="bottomMargin">
          <number>7</number>
         </property>
         <item>
          <widget class="QPushButton" name="btnDownload">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Tải video</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnPause">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 85, 0);</string>
           </property>
           <property name="text">
            <string>Tạm dừng</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnStop">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(185, 0, 0);</string>
           </property>
           <property name="text">
            <string>Hủy tải</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
